platform/src/test/java/platformfeatures/PF_Footer.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Validate footer links
+@FooterPage
+Scenario Outline: Validate Footer Links
+
+Given Navigate to URL and check footer links <browser>
+#SAnd check footer links after login
+Then Close browser
```

Comment:
Ex: 
Then validate links can be opened
And close browser

Reply:
Used And feature.

Reply:
As per above comment, I cant determine what are you validating in Then step. Please add new step

Reply:
Updated Feature file.

Reply:
Updated Feature file.